### PR TITLE
Update Internxt token info

### PR DESCRIPTION
**Pull request by fvsegarra**

### CODE REVIEW

File: tokens/eth/0xa8006C4ca56F24d6836727D106349320dB7fEF82.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "ens_address": "",
   "website": "https://internxt.com/",
   "logo": {
-    "src": "https://internxt.com/img/logos/internxtcircle.png",
+    "src": "http://internxt.com/img/icons/token.png",
```
Comment (ligi) [inline tokens/eth/0xa8006C4ca56F24d6836727D106349320dB7fEF82.json]: why downgrade from https to http?

Comment (fvsegarra) [reply tokens/eth/0xa8006C4ca56F24d6836727D106349320dB7fEF82.json]: didn't realize; will update!

**Comment by fvsegarra** [comment]
Done! :)

**Comment by ligi** [comment]
obsoleted by #330

**Pull request closed without merging**